bundles/action/org.openhab.action.telegram/README.md

```diff
@@ -42,11 +42,14 @@ The URL can be specified using the http, https, and file protocols.
 
 The action can be configured in `services/telegram.cfg`.
 
-| Property            | Default | Required | Description                           |
-|---------------------|---------|:--------:|---------------------------------------|
-| bots                |         | Yes      | Comma-separated list of `<bot-name>`s |
-| `<bot name>.chatId` |         | Yes      | chat id                               |
-| `<bot name>.token`  |         | Yes      | authentication token                  |
+| Property                | Default | Required | Description                                                  |
+|-------------------------|---------|:--------:|--------------------------------------------------------------|
+| bots                    |         | Yes      | Comma-separated list of `<bot-name>`s                        |
+| `<bot name>.chatId`     |         | Yes      | chat id                                                      |
+| `<bot name>.token`      |         | Yes      | authentication token                                         |
+| `<bot name>.parseMode`  |         | No       | Support for formatted messages, values: `Markdown` or `HTML` |
```

Comment:
This needs to specify what the default is when the parameter is not configured.

Suggest something like:

````
Message formatting to be used. Supported values: `Markdown` or `HTML`. Default is Markdown.
````

Reply:
I added a bit more text that hopefully it is better understandable that if no value is set then no formatting is used at all.